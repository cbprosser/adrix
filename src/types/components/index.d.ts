export * from './AdvancedSearchForm';
export * from './MTGStaticTypes';
export * from './MultipleCardDisplay';
export * from './SimpleCardSearchBar';
export * from './Symbol';
