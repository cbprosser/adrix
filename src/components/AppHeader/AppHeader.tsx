import { AppBar, Container, Toolbar, Typography } from '@mui/material';
import { SimpleCardSearchBar } from '../SimpleCardSearchBar';

export const AppHeader = () => {
  return (
    <AppBar position="static">
      <Container fixed>
        <Toolbar>
          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
            Adrix
          </Typography>
          <SimpleCardSearchBar hideError />
        </Toolbar>
      </Container>
    </AppBar>
  );
};
