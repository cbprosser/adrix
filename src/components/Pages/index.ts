export * from './AdvancedSearch';
export * from './Home';
export * from './Results';
