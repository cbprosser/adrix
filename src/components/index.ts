export * from './AppHeader';
export * from './CardDisplay';
export * from './Pages';
export * from './SimpleCardSearchBar';
