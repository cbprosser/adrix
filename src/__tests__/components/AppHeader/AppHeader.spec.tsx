import { render } from '@testing-library/react';
import { AppHeader } from '../../../components/AppHeader';

jest.mock('../../../components/SimpleCardSearchBar', () => ({
  SimpleCardSearchBar: (props: any) => (
    <div {...props}>SimpleCardSearchBar</div>
  ),
}));

describe('AppHeader Suite', () => {
  it('should match the snapshot', () => {
    const view = render(<AppHeader />);

    expect(view.asFragment()).toMatchSnapshot();
  });
});
