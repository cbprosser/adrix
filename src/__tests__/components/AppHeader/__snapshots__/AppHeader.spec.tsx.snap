// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader Suite should match the snapshot 1`] = `
<DocumentFragment>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg MuiContainer-fixed css-1ss2ei0-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      >
        <div
          class="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
        >
          Adrix
        </div>
        <div>
          SimpleCardSearchBar
        </div>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
